docs/Field-Mapping.md

```diff
@@ -307,13 +315,12 @@ All payment methods:
   
 The following are required only for Installment-Type Payment Methods (mainly Klarna): 
  
- * `birthdate` and `vatid` : these fields are only available on the CT Customer and not on the Cart. I.e. Guest checkouts
-   cannot do some payment methods.
-   * If the fields `dateOfBirth` (type Date) and `vatId` (type String) respectively are set as custom object on the 
-     Cart / Order and have the right type they are used and take precedence over the Fields on the Customer Object
- * `gender`: Check the Cart for a custom field `customerGender`, as fallback check the Customer for a custom field 
-    named `gender`. If the first existing is of Type `Enum` and has a value `Male` or `Female` -> use that one as `f` or `m` respectively.  
- * `ip`: Check for a custom Field `customerIPAddress` of type `String` on the CT Cart / Order. 
+ * If the fields `dateOfBirth` (type Date) and `vatId` (type String) respectively are set as custom object on the 
+     Cart / Order and have the right type they are used and take precedence over the Fields on the Customer Object 
```

Comment:
updated doc to actual behavior: 10296102729383be554c5297c5f046a37afdcc74 and 07b56df2870ff3de7f9ec4c2ae4b841387036c58